api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirPatientDaoImpl.java

```diff
@@ -41,7 +41,12 @@
 	@Inject
 	@Named("sessionFactory")
 	SessionFactory sessionFactory;
-	
+
+	@Override
+	public List<Patient> findAllPatients() {
+		return patientService.getAllPatients();
```

Comment:
Could we use the pattern `sessionFactory.getCurrentSession().createCriteria(...).list();` here?